include/seqan3/std/concepts

```diff
@@ -64,81 +64,100 @@ namespace std
 //!\addtogroup concepts
 //!\{
 
-// ============================================================================
-//  Core language concepts
-// ============================================================================
+// ==========================================
+// [concepts.lang], language-related concepts
+// ==========================================
 
+// [concept.same], concept same_as
 using ::concepts::same_as;
+
+// [concept.derived], concept derived_from
 using ::concepts::derived_from;
+
+// [concept.convertible], concept convertible_to
 using ::concepts::convertible_to;
+
+// [concept.commonref], concept common_­reference_­with
 using ::concepts::common_reference_with;
+
+// [concept.common], concept common_with
 using ::concepts::common_with;
+
+// [concept.arithmetic], arithmetic concepts
 using ::concepts::integral;
 using ::concepts::signed_integral;
 using ::concepts::unsigned_integral;
-
-/*!\interface std::floating_point <>
- * \brief The concept std::floating_point<T> is satisfied if and only if T is a floating-point type.
- * \sa https://en.cppreference.com/w/cpp/types/is_floating_point
- */
+// using ::concepts::floating_point;
 //!\cond
 template <typename t>
 SEQAN3_CONCEPT floating_point = std::is_floating_point_v<t>;
```

Comment:
why did you remove the documentation fthe floating_point concept although we still need to implement it?

Reply:
Because std::floating_point should use the external dokumentation of cppreference.